home/.vimrc

```diff
@@ -97,7 +97,7 @@ map \ :NERDTreeToggle<CR>
 
 map <Leader>p gg:s/\s*$<CR>/\d\+-<CR>ywggA [#<esc>pa]<esc>
 nmap <Leader>l iLorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.<esc>
-
+nmap <Leader>skip gg2o<esc>i[skip ci]<esc>
```

Comment:
More general point: Should vim-snippets be part of our dotfiles? I think it
needs some python on the backend, so not sure what mad puppetry that might
involve.

Barring the addition of vim-snippets, kosher to add a snippets dir to our
vim bundle directory?
-t

2015-03-12 10:52 GMT-04:00 Ben Oakes <notifications@github.com>:

> In home/.vimrc
> <https://github.com/ContinuityControl/dotfiles/pull/4#discussion_r26307920>
> :
>
> > @@ -97,7 +97,7 @@ map \ :NERDTreeToggle<CR>
> >
> >  map <Leader>p gg:s/\s*$<CR>/\d\+-<CR>ywggA [#<esc>pa]<esc>
> >  nmap <Leader>l iLorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.<esc>
> > -
> > +nmap <Leader>skip gg2o<esc>i[skip ci]<esc>
>
> I like the snippet idea.
>
> I guess I use lorem ipsum a lot more than you, too. [image: :smile:]
> Though I have no problem if we decide to snippetize that too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ContinuityControl/dotfiles/pull/4/files#r26307920>.
>



-- 
Tom Reznick
Junior Software Engineer, Continuity
914-519-8966

-- 


-- 

------------------------------
This email and all files transmitted with it are intended solely for use by 
the intended recipient.  Any other use is prohibited.  If you received this 
email in error, please notify the sender, delete the email and destroy all 
attachments.  Opinions and ideas expressed herein are solely those of the 
sender, and may not reflect the official position of Continuity.